fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -1993,6 +1993,13 @@ ACTOR Future<Void> commitProxyServerCore(CommitProxyInterface proxy,
 	state Future<Void> dbInfoChange = commitData.db->onChange();
 	//TraceEvent("ProxyInit3", proxy.id());
 
+	// Add TLog groups to collection
+	const auto& logset = commitData.db->get().logSystemConfig.tLogs[0];
+	for (const auto& gid : logset.tLogGroupIDs) {
+		TLogGroupRef group = makeReference<TLogGroup>(gid);
```

Comment:
I think should add a TODO here to also add list of workers in that group.  Can figure out from logSet iself, but in that case, we should probably refactor to not keep same thing at two places in future.

Reply:
Done.